### DISCUSSION TITLE
fan speed settings working as expected for anyone?

### DISCUSSION THREAD

**Discussion by turbolooser** [revk/xESP32-Faikout#216] [General]
Runing 3 Faikin now and the fan speed settings are driving me nuts as it seems that only auto and night do reliably change the rpm's actually. If I set the fan speed to 1,2,3,4 or 5 it does not do anything to my Perfera indoor units in most cases.
It just worked once and I collected the rpm speeds but I cannot reproduce. Just now any setting does not change anything. I am confused

In my case its
1 = 690
2 = 770
3 = 860
4 = 860
5 = 860
auto = 590
night = 520

I have tested the fan speed using the webgui and in manual mode and also faikin auto mode and there is no clear pattern to me on when certain fan speeds would work or not.
Any ideas?

I also wanna add that this seems to be the reason that I cannot reach the target temperatures as expected

**Comment by turbolooser** [comment]
example screenshot
target temp = 19.5
actual BLE temp: 18.23

Only fan buttons doing a change is auto vs night
auto = 690 rpm
night = 590 rpm
none of the other buttons (1,2,3,4,5) work or change the fanspeed. But with the given temp difference of 1.27 celsius I would expect the fan to go higher to get to the target of 19.5 but it does not.

<img width="531" alt="image" src="https://github.com/revk/ESP32-Faikin/assets/115738363/b4b97ea6-e55d-4bdf-8932-08309a97e6ea">

**Comment by turbolooser** [comment]
Switched to non-auto mode and seems like the fanspeed settings were working as mentioned in post 1 table.
Now switched back to auto-mode and still fanspeeds 1,2,3 are working 4,5 do not change the rpms (ssee post 1).
I am sure that I did that switch earlier many times and sometimes fanspeeds are working another time they are not.

At least I can run now on fanspeed 3 which should lead to target temperature as the inverter can now produce more heat which is enabled to be transported (liquid = 37)

<img width="465" alt="image" src="https://github.com/revk/ESP32-Faikin/assets/115738363/218afc0e-97b3-4820-bec9-0487ede3d77b">


btw: the fanspeed naming I think could be also tweaked to just the numbers or everything in lowercase. I think the numbers should be good enough. 👍

**Reply by revk** [reply]
Sounds odd, I can try and look in a bit.

**Reply by turbolooser** [reply]
Now again I cannot change the fanspeed on 1,2,3,4,5
Even auto or night do not work. fanspeed is static to 690 rpm

<img width="477" alt="image" src="https://github.com/revk/ESP32-Faikin/assets/115738363/5a3b8651-e1b0-402b-b115-4d72ce8ce24f">
<img width="448" alt="image" src="https://github.com/revk/ESP32-Faikin/assets/115738363/a1584b82-3b08-4ea5-bb10-3383f4295fd0">

**Reply by turbolooser** [reply]
I had to unplug the faikin as I have the feeling that its having its own life on fansettings. Trying withour faikin now and checking if daikin itself has some issues or not

update: It seems that Daikin itself is regulating the fanspeed max based on the demand control. Low demand control means only lower fanspeeds (1,2,3) are allowed. So setting this again to 100 or off seems to enable all fanspeeds. To me there is still an issue as sometimes it seems that the demand control (daikin cloud) is not correctly working and fanspeeds remain stuck.

**Reply by MuzzaMcD** [reply]
Interesting to see some of your findings on the use of Demand Control. I was considering investigating that soon. I have only recently installed some Faikin controls into my various split systems and really appreciate the efforts of @revk and the various discussions here. 👍

**Reply by MUN0X** [reply]
I'm not so sure that daikin regulates its fan speed based on the demand control setting or that it only allows certain fanspeeds depending on the demand control value. To me it seems that the fanspeed is limited by the liquid temperature and that high fan speeds are not allowed when the liquid temperature is low.
Of course you have a direct impact on the liquid temperature by enabling demand control as you reduce the amount of power you put into the system. But the # of active units on a multi-splits impacts this as well.

I wonder if this is comfort thing, preventing the unit from blowing large quantities of cold air...

**Comment by revk** [comment]
I think the fact you have it in auto control is an issue here - it sets temp high or low which impacts the actual fan speed. I.e. if I am cooling to a high temperature (which is basically saying it *off* because cool enough) the fan actually stops.

Try in non *faikin auto* mode.
